cratedb_toolkit/admin/xmover/analysis/shard.py

```
@@ -836,6 +839,158 @@
+        if go_live:
+            with Live(self.generate_shards_table(self._get_top_shards(self.latest_shards, n_shards), self.seq_deltas), refresh_per_second=4, console=console) as live_shards:
+                while True:
+                    sleep(interval_in_seconds)
+                    self.refresh_data()
+                    live_shards.update(self.generate_shards_table(self._get_top_shards(self.latest_shards, n_shards), self.seq_deltas))
+        else:
+            iterations = 0
+            while True:
+                sleep(interval_in_seconds)
+                self.refresh_data()
+                shards_table = self.generate_shards_table(self._get_top_shards(self.latest_shards, n_shards), self.seq_deltas)
```

Ruff (0.12.2)

Line too long (126 > 120)

(E501)